log4j2/src/main/java/com/google/common/flogger/backend/log4j2/Log4j2SimpleMessageFormatter.java

```diff
@@ -0,0 +1,255 @@
+package com.google.common.flogger.backend.log4j2;
```

Comment:
I don't have time for a full review, but if you are ignoring metadata, just use BaseMessageFormatter

https://github.com/google/flogger/blob/master/api/src/main/java/com/google/common/flogger/backend/BaseMessageFormatter.java#L56

There's no need to copy classes like this and it'll just add to the maintenance burden further down the line.

Reply:
I changed it and I am using Flogger's `LogMessageFormatter` and `BaseMessageFormatter` now.